docs/godot/feedback.md

```diff
@@ -40,3 +40,7 @@ func _on_pressed() -> void:
 	await Talo.feedback.send(internal_name, feedback_comment)
 	print("Feedback sent for %s: %s" % [internal_name, feedback_comment])
 ```
+
+:::tip
+If you'd like to attach additional metadata to the feedback (player props, game state, etc.), you could always append `feedback_comment` with said metadata, before giving it to the `send()` function.
```

Comment:
would just tweak the wording slightly here to make it read a little better:
"You can attach additional metadata  (player props, game state, etc.) to feedback by appending it to the `feedback_comment` parameter before calling `send()`."

Reply:
Yeah you're right, the wording wasn't good. What about :
> [...] to the `feedback_comment` String before calling `send()`.

Would emphasize the fact that you're appending to a String, and you'll probably want to typecast your metadata to String beforehand.

Reply:
yeah that works!

Reply:
@elliotfontaine any chance you can update this soon?

Reply:
Whoops sorry, completely missed the ping  :/

Reply:
no problem lets get this merged and I'll update the unity docs. thanks!